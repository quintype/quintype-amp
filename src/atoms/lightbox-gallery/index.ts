import { LightboxGallery } from "./lightbox-gallery";
export { LightboxGallery };
