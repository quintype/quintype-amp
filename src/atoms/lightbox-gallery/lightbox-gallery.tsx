import React from "react";
import { Helmet } from "react-helmet";

export const LightboxGallery = () => {
  // to be used alongwith amp-carousel or amp-img. Doesn't accept children or props
  // usage: add attribute "lightbox" on amp-img or amp-carousel
  return (
    <Helmet>
      <script
        async={undefined}
        custom-element="amp-lightbox-gallery"
        src="https://cdn.ampproject.org/v0/amp-lightbox-gallery-1.0.js"
      />
    </Helmet>
  );
};
