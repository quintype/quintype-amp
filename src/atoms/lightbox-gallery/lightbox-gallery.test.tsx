import React from "react";
import { LightboxGallery } from "./lightbox-gallery";
import { shallow } from "enzyme";

describe("LightboxGallery", () => {
  it("should render", () => {
    const wrapper = shallow(<LightboxGallery />);
    expect(wrapper).toMatchSnapshot();
  });
});
