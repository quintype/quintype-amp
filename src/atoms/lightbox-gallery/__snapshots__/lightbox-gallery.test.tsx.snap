// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LightboxGallery should render 1`] = `
<HelmetWrapper
  defer={true}
  encodeSpecialCharacters={true}
>
  <script
    custom-element="amp-lightbox-gallery"
    src="https://cdn.ampproject.org/v0/amp-lightbox-gallery-0.1.js"
  />
</HelmetWrapper>
`;
